ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -141,6 +153,17 @@ class CommandBarButton: UIButton {
         }
     }
 
+    private func addCustomView(_ view: UIView) {
+        view.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(view)
+
+        /// Constrain view to edges of the button
+        view.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
+        view.trailingAnchor.constraint(equalTo: trailingAnchor).isActive = true
+        view.topAnchor.constraint(equalTo: topAnchor).isActive = true
+        view.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
```

Comment:
Every time any constraint is activated, Apple's layout engine does some computation. This is a small amount of activations, but in practice we've seen bugs before where we've activated constraints in a loop and it was extremely slow. We were able to make non-trivial perf gains by creating the constraints in the loop and then making one call to activateConstraints after.

Again, it's only 4 so the practical impact is probably not game changing, but as best practice we should activate these all at once using NSLayoutConstraint.activateConstraints([...])